scripts/spack/packages/cray-mpich/package.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+
+from spack import *
+from spack.util.module_cmd import get_path_args_from_module_line, module
+
+
+class CrayMpich(Package):
+    """Cray's MPICH is a high performance and widely portable implementation of
+    the Message Passing Interface (MPI) standard."""
+
+    homepage = "https://docs.nersc.gov/development/compilers/wrappers/"
+    has_code = False    # Skip attempts to fetch source that is not available
+
+    maintainers = ['haampie']
+
+    # SERAC EDIT START
+    version('8.1.13')
```

Comment:
Upsteamed to Spack?

Reply:
That version is still not in Spack develop. I copied the package file from Axom. I could make this a child of the builtin package though.

Reply:
The package is now inherited from the spack builtin so its a bit smaller